### PR TITLE
Expose root privilege of the dev db.

### PR DESCRIPTION
**Pull request by skywhale**

### CODE REVIEW

File: tools/gradle-plugins/gradle-curiostack-plugin/src/main/java/org/curioswitch/gradle/plugins/gcloud/CurioDatabasePlugin.java [hunk 1]
```diff
@@ -68,8 +68,8 @@ private static void addTasks(Project project) {
     if (devAdminPassword != null) {
       generateDevDbDockerfile.environmentVariable("MYSQL_PASSWORD", devAdminPassword);
     }
-    generateDevDbDockerfile.environmentVariable("MYSQL_RANDOM_ROOT_PASSWORD", "yes");
-    generateDevDbDockerfile.environmentVariable("MYSQL_ONETIME_PASSWORD", "yes");
+    // Root privilege need to be exposed to create users other than MYSQL_USER.
+    generateDevDbDockerfile.environmentVariable("MYSQL_ROOT_PASSWORD", "root");
```
Comment (chokoswitch) [inline tools/gradle-plugins/gradle-curiostack-plugin/src/main/java/org/curioswitch/gradle/plugins/gcloud/CurioDatabasePlugin.java]: This shouldn't be such a simple password - let's go ahead and use the same devAdminPassword for the user and root.

Comment (skywhale) [inline tools/gradle-plugins/gradle-curiostack-plugin/src/main/java/org/curioswitch/gradle/plugins/gcloud/CurioDatabasePlugin.java]: I'm sorry for my slow response. Didn't have notifications set up right.

**Comment by skywhale** [comment]
PTAL

File: tools/gradle-plugins/gradle-curiostack-plugin/src/main/java/org/curioswitch/gradle/plugins/gcloud/CurioDatabasePlugin.java [hunk 2]
```diff
@@ -67,9 +67,9 @@ private static void addTasks(Project project) {
     String devAdminPassword = config.devAdminPassword();
     if (devAdminPassword != null) {
       generateDevDbDockerfile.environmentVariable("MYSQL_PASSWORD", devAdminPassword);
+      // Root privilege need to be exposed to create users other than MYSQL_USER.
```
Comment (anuraaga) [inline tools/gradle-plugins/gradle-curiostack-plugin/src/main/java/org/curioswitch/gradle/plugins/gcloud/CurioDatabasePlugin.java]: needs

**Comment by chokoswitch** [comment]
Thanks!

**Pull request merged**